src/Fixer/Comment/HeaderCommentFixer.php

```diff
@@ -278,54 +278,60 @@ private function findHeaderCommentCurrentIndex(Tokens $tokens, int $headerNewInd
      */
     private function findHeaderCommentInsertionIndex(Tokens $tokens, string $location): int
     {
-        if ('after_open' === $location) {
+        $openTagIndex = $tokens[0]->isGivenKind(T_OPEN_TAG) ? 0 : $tokens->getNextTokenOfKind(0, [[T_OPEN_TAG]]);
+
+        if (null === $openTagIndex) {
```

Comment:
this can never be true because of the candidate check I think? 
I think `openTagIndex` can always be set to `$tokens->getNextTokenOfKind(-1, [[T_OPEN_TAG]]);`